### ISSUE TITLE
Gritter doesn't work with rails 4.x

### ISSUE THREAD

**Issue by pablosalgadom** [RobinBrouwer/gritter#52]
The gem just doesn't work because in the "vendor/assets/stylesheets/gritter.css.scss" file have trouble with loading the images with the error:

"Asset filtered out and will not be served: add `Rails.application.config.assets.precompile += %w( ie-spacer.gif )` to `config/initializers/assets.rb` and restart your server"

(for the record, if i add the images into the assets.rb doesn't work either)

in the issue #38  said the solution: "image_url calls should be changed to asset-url calls".

**Comment by emn178** [comment]
Hi all,

I have a pull request #55  to fix this problem.
Before author accepts it, you can change Gemfile to this: 

````
gem 'gritter', :git => 'https://github.com/emn178/gritter.git'
````

**Comment by RobinBrouwer** [comment]
Fixed it in v1.2.0.
